src/main/java/com/github/swapbook/api/Users.java

```diff
@@ -1,22 +1,30 @@
 package com.github.swapbook.api;
 
-import com.github.swapbook.configuration.SecurityConstants;
-import io.jsonwebtoken.Jwts;
 import com.github.swapbook.model.User;
 import com.github.swapbook.repositories.users.UserDBRepository;
+import com.github.swapbook.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import javax.servlet.http.Cookie;
 import javax.servlet.http.HttpServletRequest;
 import java.util.List;
 
 @RestController
 public class Users {
-    @Autowired
+    //@Autowired
     UserDBRepository userRepository;
 
+    UserService userService;
+
+    @Autowired
+    public Users (UserDBRepository userRepository,
```

Comment:
@kolorowerowe @MarcinKozak005 czy jak konstruktor bedzie autowerid to bedzie ok ? wydaje mi się ze bedzie działąło tak samo

Reply:
Ja na ten moment nie umiem w te @Autowired, więc Ci nie pomogę na razie xD